_schedule/talks/2019-09-23-12-50-t0-lightning-talks.md

```diff
@@ -12,6 +12,13 @@ room: Salon A-E
 sitemap: true
 title: Lightning Talks
 track: t0
+video_url: https://www.youtube.com/watch?v=b5jS4mLR1N0
 ---
 
-Salon A-E
\ No newline at end of file
+- Wayne Merry - 00:00
+- Ayaz Amlani - 5:24
+- Tyler Rudie - 9:37
+- Pandy Knight - 14:30
```

Comment:
```suggestion
- Pandy Knight - [14:30](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=870)
```

Reply:
Q: Do we want to address him as Andrew, Andy, or Pandy? (In his talk, he says "I'm Andy")

![image](https://user-images.githubusercontent.com/7773256/67231353-c72a5e80-f404-11e9-9307-498ff189b7f2.png)